src/components/popup/popup.jsx

```diff
@@ -32,27 +18,24 @@ export default class Popup extends Component {
 	};
 
 	onCancel = () => {
-		this.setState({
-			selectedPanelIndex: SECTION_INTRO
-		});
+		this.props.store.section = SECTION_INTRO;
+		this.forceUpdate();
```

Comment:
Maybe we should simply use `this.props.store.updateSection(SECTION_INTRO)` in order to omit `forceUpdate()` usage.